src/components/ListTasks.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { FlatList } from 'react-native';
+import Task from './Task';
+
+const data = [
+  { title: 'task1', description: 'description of task1', id: '1' },
+  { title: 'task2', description: 'description of task2', id: '2' },
+  { title: 'task3', description: 'description of task3', id: '3' },
+];
+
+class ListTasks extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tasks: data,
+    };
+  }
+
+  render() {
+    const { tasks } = this.state;
```

Comment:
I like the way you are using destructuring here 🙂

Reply:
thanks 😄